src/main/kotlin/org/rust/cargo/toolchain/tools/Cargo.kt

```diff
@@ -202,7 +202,7 @@ class Cargo(toolchain: RsToolchainBase, useWrapper: Boolean = false)
         projectDirectory: Path,
         listener: ProcessListener?
     ): BuildMessages? {
-        val additionalArgs = listOf("--message-format", "json", "--workspace")
+        val additionalArgs = listOf("--message-format", "json", "--workspace", "--tests")
```

Comment:
Probably, it would be useful to add a comment why we use `--tests` instead of `--all-targets` just to prevent possible reader question